vra/structure.go

```diff
@@ -62,6 +64,7 @@ func indexOf(value string, items []string) (int, error) {
 	return -1, fmt.Errorf("Could not find %s in item list %v", value, items)
 }
 
+/*
```

Comment:
If you are opening another PR to remove the commented code before my VMC PR, you can leave this method commented as I'm using it in my code.

Reply:
Yes, I figured you would need it and is why I commented it rather than removed it. I wanted to pass lint so I can turn on CI/CD. I won't be making further changes until after your changes land. Please rebase your changes on top of this commit.